cves/kernel/CVE-2017-17805.yml

```diff
@@ -151,10 +163,14 @@ discovered:
 
     If there is no evidence as to how this vulnerability was found, then please
     explain where you looked.
-  answer:
-  automated:
-  contest:
-  developer:
+  answer: |
+    According to the git.kernel.org site, the vulnerability was discovered by
+    syzbot, which is an automated system that reports bugs based on results
+    from running a fuzzer on the kernel repository. This bot is specifically
+    from Google Groups.
```

Comment:
Good answer—I recommend a brief explanation of what a fuzzer is in your answer.

Reply:
Good call!